### PR TITLE
L1 muon online DQM upgrades

### PR DESCRIPTION
**Pull request by thomreis**
A few additional histograms for the L1 muon online DQM.

- Add dR, dEta, and dPhi histograms for L1 muons pairs.
- Add invariant mass plots for L1 muon pairs.
- Add dEta and dPhi vs. pT histograms for the coordinates at the vertex minus the coordinates at the muon stations.

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/L1TMonitor/src/L1TStage2uGMT.cc [hunk 1]
```diff
@@ -284,6 +351,30 @@ void L1TStage2uGMT::bookHistograms(DQMStore::IBooker& ibooker, const edm::Run&,
   ugmtMuonPhiAtVtxEmtf = ibooker.book1D("ugmtMuonPhiAtVtxEmtf", "uGMT Muon #phi at vertex for EMTF Inputs", 126, -3.15, 3.15);
   ugmtMuonPhiAtVtxEmtf->setAxisTitle("#phi at vertex", 1);
 
+  ugmtMuonDEtavsPtBmtf = ibooker.book2D("ugmtMuonDEtavsPtBmtf", "uGMT Muon from BMTF #eta_{at vertex} - #eta_{at muon system} vs p_{T}", 80, 0, 80, 50, -0.2, 0.2);
```
Comment (davidlange6) [inline DQM/L1TMonitor/src/L1TStage2uGMT.cc]: can the # of bins in these be reduced by 10x?

Comment (thomreis) [inline DQM/L1TMonitor/src/L1TStage2uGMT.cc]: That would leave the histogram with little value for the DQM. But I can probably get a factor 2.5-3 less bins without sacrificing too much information.

**Comment by dmitrijus** [comment]
+1

File: DQM/L1TMonitor/src/L1TStage2uGMT.cc [hunk 2]
```diff
@@ -5,7 +5,9 @@ L1TStage2uGMT::L1TStage2uGMT(const edm::ParameterSet& ps)
     : ugmtMuonToken(consumes<l1t::MuonBxCollection>(ps.getParameter<edm::InputTag>("muonProducer"))),
       monitorDir(ps.getUntrackedParameter<std::string>("monitorDir")),
       emul(ps.getUntrackedParameter<bool>("emulator")),
-      verbose(ps.getUntrackedParameter<bool>("verbose"))
+      verbose(ps.getUntrackedParameter<bool>("verbose")),
+      etaScale_(0.010875),
```
Comment (davidlange6) [inline DQM/L1TMonitor/src/L1TStage2uGMT.cc]: hi @thomreis - please document these magic numbers (maybe they can be taken from somewhere else?

Comment (thomreis) [inline DQM/L1TMonitor/src/L1TStage2uGMT.cc]: Hi @davidlange6 unfortunately we do not have them stored at one single place yet. Something that is on my todo list already.

**Comment by thomreis** [comment]
Hi @dmitrijus ,
after adding a comment this would need to be signed again.
Sorry for the inconvenience.

**Comment by dmitrijus** [comment]
+1

**Comment by thomreis** [comment]
@davidlange6 can this be merged at some point? The comments have been addressed for a while already.

**Comment by davidlange6** [comment]
+1

**Pull request merged**